libs/ui/src/pure/PercentDisplay/index.tsx

```diff
@@ -1,9 +1,15 @@
-const ALMOST_HUNDRED = 99.99
+import BigNumber from 'bignumber.js'
 
-export const percentIsAlmostHundred = (percent: number) => percent > ALMOST_HUNDRED && percent < 100
+const ALMOST_HUNDRED = BigNumber(99.99)
+
+export function percentIsAlmostHundred(percent: string) {
+  const p = BigNumber(percent)
```

Comment:
Why do we need `bignumber.js` here?
The value is not supposed to be big (>100)?

Reply:
The problem is not that's big, but the precision.
For example, this order `0x0e8e47601b87c3da76d4d2cddd4ddaa18e69a3024c964b94a8fa0497dba9142fc6b13d5e662fa0458f03995bcb824a1934aa895f6682c6c8`.

The difference between executed and sell amount is 20 atoms. That's a very small amount that can't be represented with default number precision. 

This is what's shown for this order when using BigNumber

![image](https://github.com/cowprotocol/cowswap/assets/43217/a9b560bc-a1d5-4bdf-9fe3-eb73708f7389)

And this is after I convert from String to Number with this code
![image](https://github.com/cowprotocol/cowswap/assets/43217/6196c908-cba1-4433-a5f8-e18c379c3120)

![image](https://github.com/cowprotocol/cowswap/assets/43217/4624f6c4-42fc-4562-8709-634cf37bb3dd)